### DISCUSSION TITLE
VRP Disjunction Pickup and Delivery

### DISCUSSION THREAD

**Discussion by rahulj2409** [google/or-tools#2675] [Routing (and legacy CP) questions]
Could someone explain what this method does:

 /// Same as AddPickupAndDelivery but notifying that the performed node from
  /// the disjunction of index 'pickup_disjunction' is on the same route as the
  /// performed node from the disjunction of index 'delivery_disjunction'.
  void AddPickupAndDeliverySets(DisjunctionIndex pickup_disjunction,
                                DisjunctionIndex delivery_disjunction);

I'm a bit confused

**Comment by Mizux** [comment]
Suppose you have two set of nodes i.e. severals options for the pickup point and same for the delivery point.
e.g.: pickup node, one among `[16, 17, 18]` and delivery node, one among `[31, 32, 33]`.
so you should have two disjunctions one for the pickup and one for the delivery.

This method is an overload of the `AddPickupAndDeliveryPoint()`
